utils/backend/mysk/createMySKClient.ts

```diff
@@ -41,7 +41,7 @@ export default async function createMySKClient(
     fetch: async <Data extends {} | unknown = unknown>(
       path: Parameters<typeof fetchMySKAPI>["0"],
       options?: Parameters<typeof fetchMySKAPI>["2"],
-    ): Promise<FetchReturn<Data> | null> =>
```

Comment:
Why is this line removed?

Reply:
Because the API will always return data in JSON even if there is an error, so `null` can be safely removed